src/aero/fsi/FSIturbine.C

```diff
@@ -240,6 +242,12 @@ fsiTurbine::setup(std::shared_ptr<stk::mesh::BulkData> bulk)
     deflectionRamp_ = &(meta.declare_field<ScalarFieldType>(
       stk::topology::NODE_RANK, "deflection_ramp"));
 
+  distanceToRoot_ = meta.get_field<ScalarFieldType>(
+    stk::topology::NODE_RANK, "distance_to_root");
+  if (distanceToRoot_ == NULL)
+    distanceToRoot_ = &(meta.declare_field<ScalarFieldType>(
```

Comment:
Is there a way we can add fields that are created only in debug mode?

Reply:
I can make it so we only use this field if it is requested. I thought I deleted this field though.